lib/ansible/module_utils/aws.py

```diff
@@ -0,0 +1,111 @@
+# This code is part of Ansible, but is an independent component.
```

Comment:
bcoca: that's what I was trying to say in the PR text. I'm trying to start a clean-sheet so old versions of py and boto (vs boto3) don't have to be supported. It's also designed to not require so much setup (variables passed around, for instance). @agaffney and @ryansb and I talked about it briefly on IRC.